CoffeeShop.playground/Sources/LocationTest.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import XCTest
+
+extension Location: Equatable {
```

Comment:
@eandreea The extension is not used on this file.

Reply:
You are right, I've haven't placed it correctly, haven't looked at this from this point of view

Reply:
Since I'm using the conformance of Location to Equatable protocol in both CoffeeShopTest and CoffeeShopAppTest, and I have to put the code only in one place, I support my first decision to put it in LocationTest. If the equality test would have been required in only one location, then I would support that the code be moved in that file

Reply:
What about having the Equatable conformance in a separated file if is needed in multiple places?

Reply:
Of course that works too, but I don't consider it necessary here. If something more complex was required, then the separate file would be best.